Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/MSTSBrakeController.cs

```diff
@@ -104,13 +105,16 @@ public override void UpdatePressure(ref float pressureBar, float elapsedClockSec
                     switch (notch.Type)
                     {
                         case ControllerState.Release:
-                            pressureBar += x * ReleaseRateBarpS() * elapsedClockSeconds;
-                            epState = -1;
-                            break;
                         case ControllerState.FullQuickRelease:
```

Comment:
Is there a benefit in merging the Release and FullQuickRelease tokens together. I would prefer them to remain separate as it make the code easier to "read", and doesn't imply that the have the same functionality.

Also does the addition of the DecreasePressure code imply that they are no longer overcharge-protected tokens?

Reply:
I'll keep them separate then.
If overcharge is present (this will only happen if Overcharge is defined and selected), it has to be eliminated really slowly to avoid application of brakes, so that's the reason for using DecreasePressure. If no overcharge token is defined, pressure will be equal or lower than MaxPressure and DecreasePressure will have no effect.

Reply:
> "I'll keep them separate then."

Thanks for that.

Have you seen the document in this post? - 

http://www.elvastower.com/forums/index.php?/topic/34527-wishes-for-improvement-of-braking-systems/page__view__findpost__p__264462

On pg 14 for the Knorr brake it is proposing an alternate token to follow the overcharging one. 

I am not sure whether anything needs to be done to the Neutral token, as it is supposed to isolate the pipe from the brake valve. However perhaps it is worth you posing a question in the thread, and thus any necessary changes could be done as part of the changes that you are adding.

Reply:
Currently Neutral position does not isolate EQ from BP, so this has to be fixed. It's pretty straightforward to do it, but it's off-topic here I think.
Which is the other token you mention?
I have posted a message in that thread.

Reply:
That was the only token.

It would be good to tidy it all up in the one go.